code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -397,8 +398,15 @@
 			H.r_store = null
 			return
 
-	src.orient2hud(user)
-	if(src.loc == user)
+
+	if(click_to_hand && loc == user)
+		for(var/obj/item/I in contents)
+			if(user.put_in_active_hand(I))
+				add_fingerprint(user)
+				return
+
+	orient2hud(user)

```

Comment:
I think this one can do before the show_to. However I'm not fully sure of this as it's done this way in all other code.
Just looks like it's updating the HUD layout that will be shown when opened